src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1184,14 +1184,32 @@ Prevent Long URL From Breaking Out of Container
     margin-right: 5px;
 }
 
-/*
-Glyphicon for success status messages
-*/
+/*//////////////////////////////////////////////////////////
+////////////////////// Status Icons //////////////////////
+//////////////////////////////////////////////////////////*/
 
-.icon-success:before {
+.icon-success:before,
+.icon-warning:before,
+.icon-danger:before,
+.icon-info:before {
     font-family: 'Glyphicons Halflings';
```

Comment:
Can you add `/*noinspection CssNoGenericFontName*/` here to suppress warnings (in IntelliJ)?